api/v1beta1/groupversion_info.go

```diff
@@ -17,4 +17,6 @@ var (
 
 	// AddToScheme adds the types in this group-version to the given scheme.
 	AddToScheme = SchemeBuilder.AddToScheme
+
+	localSchemeBuilder = &SchemeBuilder.SchemeBuilder
```

Comment:
Is this used?  there seem no references to this.

Reply:
Used in the code generated by conversion-gen.
I added comments.
https://github.com/cybozu-go/moco/pull/350/commits/e6e44e68758bac34671ce8720857c796de73ccf9

https://github.com/cybozu-go/moco/blob/4f5c0ae1595d2b591397a17b7fd8ac68133421cc/api/v1beta1/zz_generated.conversion.go#L20